index.d.ts

```diff
@@ -3068,11 +2874,9 @@ declare namespace Eris {
     createdAt: CT extends "withMetadata" ? number : null;
     guild: CT extends "withMetadata"
       ? Guild // Invite with Metadata always has guild prop
-      : CH extends Extract<InviteChannel, GroupChannel> // Invite without Metadata
-        ? never // If the channel is GroupChannel, there is no guild
-        : CH extends Exclude<InviteChannel, InvitePartialChannel> // Invite without Metadata and not GroupChanel
-          ? Guild // If the invite channel is not partial
-          : Guild | undefined; // If the invite channel is partial
```

Comment:
Bots can still fetch group invites

Reply:
Can probably just type this as InvitePartial in that case

Reply:
Wait, is this not the correct behaviour anyways? We could check for channel type but that causes more headaches for not much profit

Reply:
UnknownChannel when? 👉👈